.github/workflows/langwatch-app-ci.yml

```
@@ -0,0 +1,107 @@
+        working-directory: langwatch
+        run: npm run build
+
+      - name: Save Next.js cache
+        if: steps.next_cache.outputs.cache-hit != 'true'
+        uses: actions/cache/save@v4
+        with:
+          path: |
+            langwatch/.next/cache/webpack
+            langwatch/.next/cache/swc
+          key: ${{ steps.next_cache.outputs.cache-primary-key }}
+          compression-level: 1
```

actionlint (1.7.7)

input "compression-level" is not defined in action "actions/cache/save@v4". available inputs are "enableCrossOsArchive", "key", "path", "upload-chunk-size"

(action)